monasca_rsyslog/client.py

```diff
@@ -26,13 +29,23 @@
 cfg.CONF.register_opts([
     cfg.StrOpt('url',
                default=None,
-               help='Specify URL for the logging API')
+               help='Specify URL for the logging API'),
+    cfg.IntOpt('max_batch_size',
+               default=100,
```

Comment:
Seems like a reasonable choice. We should generally keep posts under 1MB with this.